### PR TITLE
Call softDeleteStory in StoriesTable

### PR DESCRIPTION
**Pull request by andrewyguo**
## Notion ticket link

[Ticket ](https://www.notion.so/uwblueprintexecs/a658c933d18c48d4b43fd99bda2bd3b4?v=ba22ac93d09040ff8219c60ac7caabdb&p=bc19836e41514ef5bc07c3e92da85ba0)

## Implementation description

- Call soft delete stories from story table 
- Change story_service `get_stories` function to use `Story` instead of `StoryAll`

## Steps to test

1. Delete story from stories table 
2. Make sure story no longer appears after the page automatically refreshes 

## What should reviewers focus on?

- Statistical mechanics and the anatomy of Cetaceans 

## Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] For backend changes, I have run the appropriate linters: `docker exec -it planet-read_py-backend_1 /bin/bash -c "black . && isort --profile black ."` and I have generated new migrations: `flask db migrate -m "<your message>"`
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by andrewyguo** [comment]
Oops, will look into the failing test

File: backend/python/migrations/versions/6bcd31d976d6_create_languages_table.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
 Create Date: 2021-12-31 22:23:00.847824
 
 """
-from alembic import op
```
Comment (andrewyguo) [inline backend/python/migrations/versions/6bcd31d976d6_create_languages_table.py:8]: this was the linter I think

Comment (gaoxk) [inline backend/python/migrations/versions/6bcd31d976d6_create_languages_table.py:8]: just checked - reverting this change does not make the linter unhappy. can you undo this?

**Review by gaoxk — APPROVED** [review]
works! plz address nits before merging

nit: can you delete the `backend/python/tools/insert_test_data.py.isorted` file thats committed? seems to have no use

**Pull request merged**